vscode/test/e2e/command-core.test.ts

```diff
@@ -93,3 +93,39 @@ test.extend<ExpectedV2Events>({
     // Code lens should be at the start of the function (range expanded from click position)
     await expect(page.getByText('* Mocked doc string')).toBeVisible()
 })
+
+test.extend<ExpectedV2Events>({
+    // list of events we expect this test to log, add to this list as needed
+    expectedV2Events: [
+        'cody.extension:installed',
+        'cody.codyIgnore:hasFile',
+        'cody.auth.login:clicked',
+        'cody.auth.signin.menu:clicked',
+        'cody.auth.login:firstEver',
+        'cody.auth.signin.token:clicked',
+        'cody.auth:connected',
+        'cody.command.explain:executed',
+    ],
+})('Explain Command from Commands Tab', async ({ page, sidebar }) => {
+    // Sign into Cody
+    await sidebarSignin(page, sidebar)
+
+    // Open the File Explorer view from the sidebar
+    await sidebarExplorer(page).click()
+
+    // Open the buzz.ts file from the tree view
+    await page.getByRole('treeitem', { name: 'buzz.ts' }).locator('a').dblclick()
+    await page.getByRole('tab', { name: 'buzz.ts' }).hover()
+
+    // Click on some code within the function
+    await page.getByText("fizzbuzz.push('Buzz')").click()
+
+    // Execute the command from the Commands tab in Chat view
+    await focusSidebar(page)
+
+    const chatPanel = page.frameLocator('iframe.webview').last().frameLocator('iframe')
+    // Click on the Commands view icon
+    await chatPanel.locator('[id="radix-\\:r0\\:-trigger-commands"]').getByRole('button').click()
+    await chatPanel.getByRole('button', { name: 'Explain code' }).click()
+    await chatPanel.getByText('hello from the assistant').hover()
```

Comment:
I always find this a confusing way to do a test, I'd personally find:
```suggestion
    await expect(chatPanel).toContainText('hello from the assistant')
```
or 
```suggestion
    await expect(getChatInputs(chatPanel).first()).toContainText('hello from the assistant')
```

Reply:
Ah my bad! Since this is not chatInput i just updated it to check for the Text :)